README.md

```diff
@@ -379,6 +380,11 @@ Always prefer `ReadonlyArray` over a regular `Array` unless it must be possible
 We use [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html) for our branching
 strategy.
 
+Branch names should adhere to the following structure:
+
+- `bugfix` or `feature` + `/` + `{TICKET_KEY}-{TICKET_TITLE}` e.g.
```

Comment:
We should not use the name "bugfix", because this indicates that there was a bug.  And we do not make bugs :-)

Better call this just `fix`

Reply:
Another possible branch prefix name is `hotfix`. These are branched from the `main` branch.

Reply:
Moved discussion to issues